BackendApartmentReservation/BackendApartmentReservation/Apartments/ApartmentRepository.cs

```diff
@@ -61,5 +61,17 @@ public async Task DeleteRoomReservation(DbRoomReservation roomReservation)
             _db.DbRoomReservations.Remove(roomReservation);
             await _db.SaveChangesAsync();
         }
+
+        public async Task<int> GetNumberOfAvailableApartmentsRooms(DateTimeOffset dateFrom, DateTimeOffset dateTo)
+        {
+            var availableRooms = (from room in _db.ApartmentRooms
+                let isAvailable = !_db.DbRoomReservations.Where(r => r.Room.Id == room.Id)
```

Comment:
I would suggest to make it a common functionality together with Ieva's code.

Reply:
For example, `GetAvailableRooms` which returns `IEnumerable<DbApartmentRoom>`

Reply:
Also, this could be simplified into 
```
DbRoomReservations
    .Include(r => r.Room)
    .Where(...)
    .Select(r => r.Room)
    .ToList()
```